Workbooks/Workloads/Onboard MySQL Insights/Onboard MySQL Insights.workbook

```diff
@@ -215,17 +208,31 @@
                         "label": "MySQL Telegraf config",
                         "type": 1,
                         "isRequired": true,
-                        "value": "[[inputs.mysql]]\r\n  servers = [\"telegraf:password@tcp(127.0.0.1:3306)/\"]\r\n\t\r\n  gather_table_schema = false\r\n  gather_process_list = true\r\n  gather_user_statistics = true\r\n  gather_info_schema_auto_inc = false\r\n  gather_innodb_metrics = false\r\n  gather_slave_status = true\r\n  gather_binary_logs = false\r\n  gather_global_variables = false\r\n  gather_table_io_waits = false\r\n  gather_table_lock_waits = false\r\n  gather_index_io_waits = false\r\n  gather_event_waits = false\r\n  gather_file_events_stats = false\r\n  gather_perf_events_statements = false\r\n\r\n  ## the limits for metrics form perf_events_statements\r\n  perf_events_statements_digest_text_limit = 120\r\n  perf_events_statements_limit = 250\r\n  perf_events_statements_time_limit = 86400\r\n\r\n  ## Some queries we may want to run less often (such as SHOW GLOBAL VARIABLES)\r\n  ##   example: interval_slow = \"30m\"\r\n  # interval_slow = \"15m\"\r\n\r\n  ## Optional TLS Config (will be used if tls=custom parameter specified in server uri)\r\n  # tls_ca = \"/etc/telegraf/ca.pem\"\r\n  # tls_cert = \"/etc/telegraf/cert.pem\"\r\n  # tls_key = \"/etc/telegraf/key.pem\"\r\n  ## Use TLS but skip chain & host verification\r\n  # insecure_skip_verify = false\r\n\r\n  metric_version = 2\r\n\r\n[[inputs.cpu]]\r\n  percpu = true\r\n  totalcpu = true\r\n  collect_cpu_time = false\r\n  report_active = false\r\n\r\n[[inputs.mem]]\r\n\r\n[[inputs.netstat]]\r\n\r\n[[inputs.disk]]\r\n\r\n[[inputs.diskio]]",
```

Comment:
Only change here is "password" to "<password>" to highlight to the user, that this needs to be changed.